src/G2/Language/Naming.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE InstanceSigs #-}
```

Comment:
Don't think we need this?

Reply:
This is not the final version. I wass gonna push the final version after our meeting. I am not expecting you to check this right now.